### DISCUSSION TITLE
Brunch on a Lenovo 100e 81CY (1st-gen)

### DISCUSSION THREAD

**Discussion by iblowmymind** [sebanc/brunch#974] [Q&A]
Hello,

I recently managed to get my hands on a Lenovo 100e 81CY netbook (which is considered "1st-gen", which has an Intel Celeron N3350 (1.1 GHz base), 4 GB of RAM, 64 GB of eNAND storage, and Windows 10 pre-installed. I was thinking if I could get Brunch working on it, so I searched up the board model and to my amazement found out that a Lenovo 100e Chromebook existed which was the same (except a different outer shell design) but with Chrome OS instead of Windows (I'm talking about the 1st gen model, the 2nd gen one is a bit more different). The image for that is "coral", so is it better for me to use the generic rammus or coral in this case? Thanks!

**Comment by kerta1n** [comment]
Yes, go with coral. I myself, have a Windows Inspiron 3542 laptop which has a Celeron N3050, but used the Setzer recovery, however I will probably be moving to Octopus. The main reason for this is because of [Google's ChromeOS AUP](https://support.google.com/chrome/a/answer/6220366), which states how long a certain device will get updates. Even though we are still installing ChromeOS on a different computer, we are using a recovery image for an offical Chromebook which still has an expiry dates on its updates. Coral is fine too as it is supported until June 2026, and the best way to pick a recovery image is to see how close it is in spec to your computer but still has a far-away AUP.

**Comment by iblowmymind** [comment]
Thank you! I went with coral, and everything worked fine.. until I upgraded to ChromeOS 89 + Brunch 89 (using the recovery image, not enable_updates), the touchpad sometimes creates issues (it won't register the clickpad left button unless a finger is on the touchpad itself, even if it's standing still). Is there any solution or kernel flags to solve this?

Also, let’s say I installed brunch on a USB drive, logged in, installed some apps, linux etc. If i run the built-in installer inside the Brunch chrome os to install onto the hard drive, will my personal data be carried over, or will it start fresh? Also, the USB and internal hard disk are the same size (64 GB), so should i just dd over the usb to the hard drive if installing with preferences copied over doesn’t work?

**Reply by kerta1n** [reply]
Hi, sorry for such a late response as I have had to deal with some personal stuff. I don't really get the question about the touchpad, but you could try the alt_touchpad_config flag. If you run the installer, no, your data will not transfer over, as the installer script just runs the transfer that moves data needed to boot ChromeOS. You could get a USB3.0/3.1 drive for installing ChromeOS, and then get another flashdrive and put your data on there.

**Reply by iblowmymind** [reply]
It's alright! I've tried the alt_touchpad_config flags (both of them) but they changed nothing. I sadly don't have the device with me anymore, so can't really do further testing for now.

**Comment by simplyrajatgupta** [comment]
This will work tried on lenovo laptop
**[https://github.com/sebanc/brunch/issues/1088]()**
